.studiorc

```diff
@@ -8,25 +8,22 @@
 #
 
 function install_from_source() {
-  pushd /src > /dev/null
-    build .
-    source results/last_build.env
-    hab pkg install "results/$pkg_artifact"
-    source "$(hab pkg path $pkg_ident)/bin/studio-common"
-    OPTS="--binlink" install alasconnect/shellcheck
-  popd > /dev/null
+  cd /src || return 1
```

Comment:
Why using `cd` instead of `pushd`?

Reply:
shellcheck linter https://github.com/koalaman/shellcheck/wiki/SC2164

Reply:
Is the intention to have the current working directory always be `/src` after returning from `install_from_source`? If not, `pushd`/`popd` can be useful, or I also like this idiom:
```bash
function install_from_source() {
  ( cd /src || return
    build .
    # shellcheck disable=SC1091
    source results/last_build.env
  )
  hab pkg install "results/$pkg_artifact"
  source "$(hab pkg path "$pkg_ident")/bin/studio-common"
  OPTS="--binlink" install alasconnect/shellcheck
}
```

This ensures that `install_from_source` won't have any effect on the caller's working directory. But if you want the `source results/last_build.env` to affect the caller's environment (I'm assuming you don't), you'll need to take a different approach.